README.md

```diff
@@ -1 +1,35 @@
-# Portfolio
\ No newline at end of file
+# Hello Microverse
+> This is a Practical Project, which was coded. 
+
+## Built With
+HTML5
+## Major languages
+HTML
+CSS
+
+
+### Prerequisites
+### Setup
+### Install
+### Usage
+### Deployment
```

Comment:
- [ ] Please, consider removing not using sections from your README file. It will be better, if you will add some information about how to set up your project on the local machine, for example, using the `git clone` command.